package/batocera/core/batocera-configgen/configgen/configgen/generators/fsuae/fsuaeGenerator.py

```diff
@@ -71,7 +71,7 @@ def generate(self, system, rom, playersControllers, gameResolution):
                 if (d.endswith("ipf") or d.endswith("adf") or d.endswith("dms") or d.endswith("adz")):
                     diskNames.append(name)
 
-            print("Amount of disks in zip " + str(len(diskNames)))
+            print(("Amount of disks in zip " + str(len(diskNames))))
```

Comment:
A more python 3 way of doing this is with an f-string:

```python
print(f'Amount of disks in zip {len(diskNames)}')
```

or simply

```python
print('Amount of disks in zip', len(diskNames))
```

Reply:
Fixed